modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -2,6 +2,7 @@
 {{>partial_header}}
 from __future__ import absolute_import
 
+from decimal import Decimal
```

Comment:
Please remove this per the above comment

Reply:
Done

Reply:
Can you resolve this conversation?